edx-platform/pearson-vue-theme/lms/templates/ccx/enrollment.html

```diff
@@ -86,11 +86,16 @@ <h2 class="hd hd-2">${_("Enrollment")}</h2>
         <h2 class="hd hd-2">${_("Student List Management")}</h2>
         %if messages:
           <label for="ccx_std_list_messages" class="sr">${_("CCX student list management response message")}</label>
-          <div id="ccx_std_list_messages" tabindex="-1" class="request-response-error">
+          <div id="ccx_std_list_messages" tabindex="-1" class="request-response">
             %for message in messages:
-              ${message}
+              %if message.tags == "error":
+                <p style="color:red;"> ${message}</p>
+              %elif message.tags == "success":
+                <p style="color:green;"> ${message}</p>
+              %endif
```

Comment:
```suggestion
              <div class="msg msg-${message.tags}" style="margin: 0;">
                <div class="msg-content">
                  ${message}
                </div>
              </div>
```
Same as previous suggestion but for VUE theme